### PR TITLE
[FIX] Change to Linear for upsampling

### PR DESCRIPTION
**Pull request by mattcieslak**
Addresses #819

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -87,7 +95,7 @@ your output to at least 1.3mm resolution. By choosing this resolution here,
 it means your data will only be interpolated once: head motion correction,
 susceptibility distortion correction, coregistration and upsampling will be
 done in a single step. If your are upsampling your data by more than 10%,
```
Comment (tsalo) [inline docs/quickstart.rst:97]:
```suggestion
done in a single step. If you are upsampling your data by more than 10%,
```

**Comment by tsalo** [comment]
Why do you use Lanczos windowed sinc for downsampling and upsampling to >90% of the original voxel size. Would it make more sense to just use linear no matter what?

**Comment by mattcieslak** [comment]
> Why do you use Lanczos windowed sinc for downsampling and upsampling to >90% of the original voxel size. Would it make more sense to just use linear no matter what?

Lanczos Sinc is very accurate if certain conditions are met. As long as there aren't sharp cutoffs between high and low intensity and the image resolution in the output isn't higher than the input data it is a great default (and is the default in fmriprep). A added the 90% as some wiggle room for weird voxel sizes.

Linear intepolation ends up smoothing the data, which is not something we want to do unintentionally. I think it makes sense for upsampling, since it won't introduce the ringing effects like a Sinc interpolator will. 

I didn't think BSpline would add ringing (particularly the relatively simple BSpline[3] we were using), but in some cases it looks like can. Although, this may only happen if there is significant upsampling. I'm not sure what the native resolution was of the data that showed the ringing artifacts.

**Review by cookpa — APPROVED** [review]
LGTM

**Review by tsalo — APPROVED** [review]

**Pull request merged**